nais.yaml

```diff
@@ -39,6 +39,12 @@ spec:
   prometheus:
     enabled: false
 
+  envFrom:
+    - secret: gitpull-config
+  filesFrom:
+    - secret: gitpull-key-private
+      mountPath: /enonic-xp/home/config/ssh
+
   env:
     - name: VHOST_HOST
       value: ssbno-{{branch}}.external.test.ssb.cloud.nais.io
```

Comment:
Should these urls be `https://ssbno-{{branch}}.intern.test.ssb.no`?

Reply:
yesss